### PR TITLE
Flask: add template to create box website system

### PR DESCRIPTION
**Pull request by Randium**
The current system is as basic as it can be, but the back end should be completely finished! Go check it out!

### CODE REVIEW

File: interpretation/ww_head.py [hunk 1]
```diff
@@ -154,6 +154,22 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
     if isGameMaster == True:
         help_msg += "\n__Game Master commands:__\n"
 
+        '''forcesignup'''
```
Comment (romangraef) [inline interpretation/ww_head.py]: what does this have to do with the website?

File: requirements.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ discord.py
 emoji
 https://github.com/Rapptz/discord.py/archive/rewrite.zip
 peewee
+Flask
```
Comment (Randium) [inline requirements.txt]: I don't think we need any other requirements. One can simply install Flask and work with it, right?

File: interpretation/ww_head.py (hunk 1 above)
Comment (Randium) [reply interpretation/ww_head.py]: Not much. I still had this function written and merged it to the virtual machine already, but the common repository didn't have this yet.

...you could sync with the virtual machine first, if you want. This update should then disappear from this specific merge request.

**Comment by Randium** [comment]
I moved the folders 'n' resolved the requests. Are we cool now? If we merge it to the main `website` branch, @Bentechy66 can start implementing the Ajax part, and I could work on the back end simultaneously to get the function to work.

File: communication/website/website.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
+from flask import Flask, render_template, redirect, url_for, request
+import management.boxes as box
+import random
+
+
+app = Flask(__name__)
```
Comment (romangraef) [inline communication/website/website.py]: Please use a Blueprint for it. basically replace app with an instance of `BluePrint`.. you can register a blueprint in any app with a different prefix to compose apps accross multiple files

**Comment by Randium** [comment]
Closing this pull request as it is completely covered by #138 by now.
